game/src/game/game_player.rs

```diff
@@ -202,6 +209,14 @@ pub(crate) fn play_song(
             janggu_state_with_tick.update(input_now, tick_now);
             let new_accuracies = timing_judge.judge(&janggu_state_with_tick, tick_now as u64);
 
+            // play hit sound when use git janggu
+            if janggu_state_with_tick.궁채.is_keydown_now {
+                audio_manager.play(kung_sound_data.clone()).unwrap();
+            }
+            if janggu_state_with_tick.열채.is_keydown_now {
+                audio_manager.play(deok_sound_data.clone()).unwrap();
```

Comment:
(Not important) Prefer using `.expect()` instead of `.unwrap()` unless it's absolutely clear that exceptions never happen.